tests/zeromq.terminal/Program.cs

```diff
@@ -28,12 +27,24 @@ static Program()
 
                 ["pubsub"] = PubSubScenarios.SimplePubSub,
                 ["subcancel"] = PubSubScenarios.PubSubWithCancellation,
+
+                [DESCRIBE] = Describe
             };
         }
 
+        const string DESCRIBE = "describe";
```

Comment:
Perhaps `help` is a more common name for this CLI command?

Reply:
good point